_episodes/20-parallel-raster-computations.md

```diff
@@ -0,0 +1,344 @@
+---
+title: "Parallel raster computations using Dask"
+teaching: TODO
+exercises: TODO
+questions:
+- "TODO"
+objectives:
+- "Profile the timing of your raster calculations."
+- "Open raster data as a chunked array."
+- "Recognize good practices in selecting proper chunk sizes."
+- "Setup raster calculations that take advantage of parallelization."
+keypoints:
+- "TODO"
+---
+
+Very often raster computations involve applying the same operation to different pieces of data. Think, for instance, to
+the "pixel"-wise sum of two raster datasets, where the same sum operation is applied to all the matching grid-cells of
+the two rasters. This class of tasks can benefit from chunking the input raster(s) into smaller pieces: operations on
+different data blocks can be run in parallel using multiple computing units (e.g., multi-core CPUs), thus potentially
+speeding up calculations. In addition, working on chunked data can also lead to smaller memory footprints, since one
+may bypass the need to store the full dataset in memory by processing it chunk by chunk.
+
+In this episode, we will introduce the use of Dask in the context of raster calculations. Dask is a Python library for
+parallel and distributed computing that provides a framework to work with different data structures, including chunked
+arrays (Dask Arrays). Dask is well integrated with (`rio`)`xarray` objects, which can use Dask arrays as underlying
+data structures.
+
+> ## More Resources on Dask
+>
+> TODO: Dask and Dask Arrays, with links
```

Comment:
should we just link to this here? https://docs.dask.org/en/stable/array.html

Reply:
Added an extra link to Xarray&Dask as well!